the-exterminators/error-annihilator
# SP 77 Adjust Existing Views

- Updated the favicon
- changed Assigned ticket view to show ticket number and project in the grid
- changed assigned ticket editor to bind to ticket number and project when viewing a single ticket
- assigned tickets changed certain fields into predefined lists
- User management editor changed user role into dropdown
- Also edited ticket entity
- Created Ticket Project entity
- Daniel was told about the entity changes as they are only a temporary fix so I could get my views done

## Linked Issues

## PR Description

## Changes


## How to test

1. fetch branch
2. Open, build and run project
3. Check if project builds and looks correct on first sight
4. Check the favicon on the Error Annihilator Tab: it should be our Logo now
5. Check Assigned Ticket View: You should now be able to additionally filter by ticket number and project
6. Click on a single ticket in the grid: You should now see the correct number and project displayed. There should also be predefined dropdown lists for type, status and project
7. Go to the user management view and click on a user: You should now have a predefined dropdown list of user roles
8. Some goes for creating a new user via the button
9. Go to the help view: There should be some stuff filled out now

### Expected behaviour:

## Checklist author


## Checklist reviewer/test-engineer
